sdk/astro/arm-astro/README.md

```diff
@@ -0,0 +1,109 @@
+# Azure AstroManagement client library for JavaScript
+
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure AstroManagement client.
+
+
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/astro/arm-astro) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/arm-astro) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-astro) |
```

Comment:
we should put preview link here and remove the stable link from ignore-links.txt